docs/cloud-workload-security/agent_expressions.md

```diff
@@ -0,0 +1,554 @@
+| `capset` | Process | A process changed its capacity set | 7.27 |
+| `chmod` | File | A file’s permissions were changed | 7.27 |
+| `chown` | File | A file’s owner was changed | 7.27 |
+| `exec` | Process | A process was executed or forked | 7.27 |
+| `link` | File | Create a new name/alias for a file | 7.27 |
+| `mkdir` | File | A directory was created | 7.27 |
+| `open` | File | A file was opened | 7.27 |
+| `removexattr` | File | Remove extended attributes | 7.27 |
+| `rename` | File | A file/directory was renamed | 7.27 |
+| `rmdir` | File | A directory was removed | 7.27 |
+| `selinux` | Kernel | An SELinux operation was run | 7.30 |
+| `setgid` | Process | A process changed its effective gid | 7.27 |
+| `setuid` | Process | A process changed its effective uid | 7.27 |
+| `setxattr` | File | Set exteneded attributes | 7.27 |
+| `unlink` | File | A file was deleted | 7.27 |
+| `utimes` | File | Change file access/modification times | 7.27 |
+
+## Operators
+SECL operators are used to combine event attributes together into a full expression. The following operators are available:
+
+| SECL Operator         | Types            |  Definition                           | Agent Version |
+|-----------------------|------------------|---------------------------------------|---------------|
+| `==`                  | Process          | Equal                                    | 7.27          |
+| `!=`                  | File             | Not equal                                | 7.27          |
+| `>`                   | File             | Greater                                  | 7.27          |
+| `>=`                  | File             | Greater or equal                         | 7.27          |
+| `<`                   | File             | Lesser                                   | 7.27          |
+| `<=`                  | File             | Lesser or equal                          | 7.27          |
+| `!`                   | File             | Not                                      | 7.27          |
+| `^`                   | File             | Binary not                               | 7.27          |
+| `in [elem1, ...]`     | File             | Element is contained in list             | 7.27          |
+| `not in [elem1, ...]` | File             | Element is not contained in list         | 7.27          |
+| `[~pattern, ...]`     | File             | Regex pattern is (not) contained in list | 7.27          |
+| `=~`                  | File             | String matching                          | 7.27          |
+| `&`                   | File             | Binary and                               | 7.27          |
+| `|`                   | File             | Binary or                                | 7.27          |
+| `&&`                  | File             | Logical and                              | 7.27          |
+| `||`                  | File             | Logical or                               | 7.27          |
```

Comment:
All of this should be replaced with Auto-generated content in the "Event Types" section

Reply:
I'm not sure I understand, the trigger part is auto-generated based on the different event types available

Reply:
Want to confirm: is anything on this page auto-generated? It's showing the same content as what is listed in the file here.

Reply:
Yes `docs/cloud-workload-security/agent_expressions.md` and `docs/cloud-workload-security/backend.md` are completely generated